services/common/snmp/update_pg/update_rsu_security.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import common.pgquery as pgquery
+import common.snmp.ntcip1218.rsu_security_expiration as ntcip1218_security
+from common.snmp.update_pg.update_pg_snmp import UpdatePostgresSnmpAbstractClass
+from datetime import datetime, timezone
+from multiprocessing import Pool, cpu_count
+
+
+class UpdatePostgresRsuSecurity(UpdatePostgresSnmpAbstractClass):
+    """
+    UpdatePostgresRsuSecurity is a class that manages the synchronization of RSU SCMS health expiration collected via SNMP
+    between the CV Manager PostgreSQL database and the RSUs. It provides methods to fetch configurations directly from
+    RSUs in order to insert and update SNMP configurations stored in PostgreSQL.
+    """
+
+    def insert_config_list(self, snmp_config_list):
+        """
+        Inserts a list of SNMP RSU SCMS expiration timestamps into the PostgreSQL database.
+        """
+        if len(snmp_config_list) == 0:
+            logging.info("No RSU SCMS data to insert into PostgreSQL")
+            return
+
+        query = (
+            "INSERT INTO public.scms_health("
+            "timestamp, health, expiration, rsu_id) "
+            "VALUES"
+        )
+
+        for snmp_config in snmp_config_list:
+            query += f" ('{snmp_config['timestamp']}', '{snmp_config['health']}', '{snmp_config['expiration']}', {snmp_config['rsu_id']}),"
```

Comment:
suggestion: parameterize to avoid sql injection issues

Reply:
Addressed.